.github/scripts/build-macOS.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+if [[ "${GITHUB_REF_TYPE}" == "tag" ]]; then
+    BUILD_TAG=${GITHUB_REF_NAME}
+else 
+    BUILD_TAG=$(git rev-parse --short HEAD)
+fi
+
+# install deps
+brew install SDL2
```

Comment:
On Linux and Macos, we implicitly assume people have SDL2 somehow installed.
Should we add `libSDL2.(dylib|so)` to the archive?
In that case, we might need to fix rpaths.

Food for thoughts, and for a later pr.

Reply:
yep - I ran into this problem on a different laptop where libsdl2 was not in the expected place. Definitely an issue to fix - I think as part of generating a fat `.app` package for macos